lib/semian/lru_hash.rb

```diff
@@ -34,7 +35,23 @@ def locked?
     end
   end
 
-  def initialize
+  # Create an LRU hash
+  #
+  # Arguments:
+  #   +max_size+ The maximum size of the table
+  #   +min_time+ The minimum time a resource can live in the cache
+  #
+  # Note:
+  #   The +min_time+ is a stronger guarantee than +max_size+. That is, if there are
+  #   more than +max_size+ entries in the cache, but they've all been updated more
+  #   recently than +min_time+, the garbage collection will not remove them and the
+  #   cache can grow without bound. This usually means that you have many active
+  #   circuits to disparate endpoints (or your circuit names are bad).
+  #   If the max_size is 0, the garbage collection will be very aggressive and
+  #   potentially computationally expensive.
+  def initialize(max_size: MAXIMUM_SIZE_OF_LRU, min_time: MINIMUM_TIME_IN_LRU)
```

Comment:
I would say that this should be a global configuration option on `Semian`, documented in the README. `Semian.maximum_lru_size = 1000`, with this as the default when you call this class method.

Reply:
Done.
I made `Semian.maximum_lru_size` and `Semian.minimum_lru_time` global configuration options.